project/dexter-server/public/AdminSE/adminSECtrl.js

```diff
@@ -0,0 +1,85 @@
+adminSEApp.controller('adminSECtrl', function($scope, $http, $location ,$routeParams, $log) {
+    "use strict";
+
+    $('select').on('select2:select', function (e) {
+        addModulePathList(e);
+        changeSelectedModuleCount();
+    });
+
+    $('select').on('select2:unselect', function (e) {
+        removeModulePathList(e);
+        changeSelectedModuleCount();
+    });
+
+    function changeSelectedModuleCount(){
+        var count = $scope.selectedModulePathList.length;
+        $("#removeModuleCount").html(parseInt(count));
+        if(count > 0) {
+            $("#isModuleSelected").removeClass("fa fa-folder").addClass("fa fa-folder-open");
+        }
+        if(count==0){
+            $("#isModuleSelected").removeClass("fa fa-folder-open").addClass("fa fa-folder");
+        }
+    }
+
+    function addModulePathList(evt){
+        if (!evt) { return; }
+        JSON.stringify(evt.params, function (key, value) {
+            $scope.selectedModulePathList.push(value.data.id);
+        });
+    }
+
+    function removeModulePathList(evt){
+        if (!evt) { return; }
+
+        JSON.stringify(evt.params, function (key, value) {
+            $scope.selectedModulePathList.pop(value.data.id);
+        });
+    }
+
+    var init = function(){
+        $scope.modulePathList = [];
+        $scope.selectedModulePathList = [];
+        $(".select2").select2({});
+        $('select').trigger('change.select2'); // Notify only Select2 of changes
+    };
+
+    init();
+
+    var getModulePathListUrl ='/api/v2/adminSE/modulePath';
```

Comment:
Does only adminSe get module path list?
If not, this API should be changed: /api/v2/module-path